hack/build/run-lint-checks.sh

```diff
@@ -20,11 +20,25 @@
 source hack/build/config.sh
 source hack/build/common.sh
 
+# Temporary var for pkgs that are ready to enforce linting
+LINTABLE=(pkg/controller pkg/apis pkg/client pkg/controller pkg/expectations pkg/image pkg/importer pkg/lib pkg/util)
```

Comment:
Nit, butr just fyi: `LINTABLE` can be a simple string with space separators. Then the `for` loop below is just `for p in "$LINTABLE ; do`

Reply:
Oh nice!  Thanks, much easier to remember than the crazy array syntax for bash too as a bonus.